types/priv_validator/unencrypted.go

```diff
@@ -1,66 +0,0 @@
-package types
-
-import (
-	"fmt"
-
-	crypto "github.com/tendermint/go-crypto"
-	"github.com/tendermint/tendermint/types"
-	cmn "github.com/tendermint/tmlibs/common"
-)
-
-//-----------------------------------------------------------------
-
-var _ types.PrivValidator2 = (*PrivValidatorUnencrypted)(nil)
-
-// PrivValidatorUnencrypted implements PrivValidator.
```

Comment:
where did this code go?

Reply:
This was split out from PrivValidatorFS, what is now called FilePV.  So this is what happened:

# Before, these implemented PrivValidator interface:
* types/PrivValidatorFS
* types/priv_validator/PrivValidatorJSON { embeds: PrivValidatorUnencrypted }.
* types/priv_validator/SocketClient

# Now, these implement PrivValidator interface:
* types/MockPV
* types/priv_validator/FilePV
* types/priv_validator/SocketPV

The distinction between PrivValidatorUnencrypted and PrivValidatorJSON is not as nice as having one single implementation called FilePV, which internally stores to a file via JSON.  We can even abstract out what a "File" is, e.g. we could pass in a tmlibs/DB instead, or perhaps we should call a "File" what is a currying of a tmlibs/DB and a. key.  Anyways, I had a hard time understanding what Unencrypted/JSON were and came to this realization, and since I had to fix wire anyways I figured I'd make my life easier by suggesting this change in tandem.

We can of course discuss this in another issue, but I suggest we merge this in first for brevity.  The previous authors for unencrypted+json were @ebuchman and @xla so I'll notify them.